interface/src/ui/OverlayConductor.cpp

```diff
@@ -96,24 +97,24 @@ void OverlayConductor::update(float dt) {
         _hmdMode = false;
     }
 
-    bool shouldRecenter = false;
+    bool initiateRecenter = false;
+    if (_hmdMode && headNotCenteredInOverlay()) {
+        initiateRecenter = true;
+    }
 
-    if (_suppressedByHead) {
-        if (updateAvatarIsAtRest()) {
-            _suppressedByHead = false;
-            shouldRecenter = true;
-        }
-    } else {
-        if (_hmdMode && headOutsideOverlay()) {
-            _suppressedByHead = true;
-        }
+    bool shouldRecenter = false;
+    if (initiateRecenter || _suppressedByHead) {
+        _suppressedByHead = !updateAvatarIsAtRest();
+        shouldRecenter = !_suppressedByHead;
```

Comment:
this logic seems very slightly different than the old code: if _suppressedByHead is false, the old code wouldn't call updateAvatarIsAtRest().  is that ok?

also this code is a little confusing with both initiateRecenter and shouldRecenter, but other than that looks good to me

Reply:
The logic had to change as part of fixing the HUD content displaying only every second HUD recentering. Not calling 

`initiateRecenter` is used to signal that the HUD should be recentered because the head is not centered in the HUD.
But updating the HUD position is suppressed if the head is moving, until such time as the head stops moving.

The new code is more straightforward than the old code, I believe.

Reply:
👍